atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/lock/SimpleLockResource.java

```diff
@@ -60,9 +61,9 @@ public boolean lockUsingLegacyLockApi(int numLocks, int descriptorSize) {
                 .builder(generateDescriptorMap(numLocks, descriptorSize))
                 .build();
         try {
-            LockRefreshToken response = transactionManager.getLockService().lock("test", request);
+            HeldLocksToken response = transactionManager.getLockService().lockAndGetHeldLocks("test", request);
```

Comment:
This is a behaviour change, but it should be fine for existing tests. I guess this was done to reuse existing tests to test the pure dialogue methods?

Reply:
Yep, this now goes through the Dialogue methods as you say. This is only in the ETE server, as well